data.json

```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "name": "Hareer Sakhi",
+    "githubUsername": "HareerSakhi",
+    "imageUrl": "https://user-images.githubusercontent.com/70807684/126065573-860e71e9-0dce-47fd-bd99-3ad77c9d7c0c.png",
+    "issueNumber": 594
+  },
```

Comment:
New profiles should be added to the end of the readme, maybe you can open a new pr and adjust this